packages/components/psammead-media-indicator/src/mediaIcons.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import styled from 'styled-components';
-import { C_EBON } from '@bbc/psammead-styles/colours';
 import { GEL_SPACING_HLF } from '@bbc/gel-foundations/spacings';
 
+// `currentColor` has been used to address high contrast mode in Firefox.
```

Comment:
Can we please update this comment so it's correct? `currentColor` has been used so that this can be seen when a user changes colour preferences in Firefox.